SonarQube.TeamBuild.PreProcessor/Roslyn/Model/AdditionalFile.cs

```diff
@@ -25,7 +25,7 @@ public class AdditionalFile
         /// <summary>
         /// The content of the file
         /// </summary>
-        [XmlAnyElement]
-        public XmlElement Content { get; set; }
+        [XmlText(DataType = "base64Binary")]
+        public byte[] Content { get; set; }
```

Comment:
Changed the profile format to match the updated spec i.e. the additional file content is written as a base-64 encoded string.